test/cfg/std.cpp

```diff
@@ -2469,21 +2471,21 @@ void uninitvar_ldexp(void)
 void invalidFunctionArg_lgamma(float f, double d, long double ld)
 {
     (void)lgamma(d);
-    // cppcheck-suppress invalidFunctionArg
+    // TODO cppcheck-suppress invalidFunctionArg
```

Comment:
This was added in #5845 and was never detected. @orbitcowboy

Reply:
Looks like `!0.0:` doesn't work, e.g. `log()` has `<valid>4.94066e-324:</valid>`.